### PR TITLE
feat(transformer): added disclaimer

### PR DESCRIPTION
**Pull request by pennfranc**
- Added a disclaimer describing the current limitations of our Transformer model with regard to the `tgt` variable that serves as input for `torch.nn.Transformer`.

- Removed a line that creates the tgt mask, since it is not currently being used.

### CODE REVIEW

**Review by LeoTafti — APPROVED** [review]

**Comment by LeoTafti** [comment]
Closing this since I branched of from this PR and solved docs missing in Sphinx for decorated functions.
